public/src/components/manual-grading/Review.tsx

```diff
@@ -20,7 +20,7 @@ interface ReviewPageProps {
 interface ReviewPageState {
     review: Review | undefined;
     open: boolean;
-    benchmarks: GradingBenchmark[];
+    // benchmarks: GradingBenchmark[];
```

Comment:
Let's remove rather than comment these lines, or is there a reason to keep them?

Reply:
Yes, I'll remove all the commented parts once everything is debugged. It is still a draft PR.